src/cmd/completions.go

```diff
@@ -0,0 +1,81 @@
+package cmd
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+var completionsNoDesc bool
+
+var completionsCmd = &cobra.Command{
+	Use:   "completions [bash|zsh|fish|powershell]",
+	Short: "Generates auto-completion scripts for Bash, zsh, fish, and PowerShell",
+	Long: `Generates auto-completion scripts for Bash, zsh, fish, and PowerShell
+
+With completions enabled, git-town TAB will show you all possible
+subcommands. As a bonus, some shells even show the short help text next to it.
+
+To enable completions:
+
+Bash:
+
+$ source <(git-town completions bash)
+
+Persist and autoload on each session:
+
+Linux: $ git-town completions bash > /etc/bash_completion.d/git-town
```

Comment:
I'm totally against this, this absolutely pollutes `.bashrc` (if someone was to keep it clean).
Furthermore, on updating completions, since it's appending, it probably would result in unpredictable behavior. Unfortunately, there seems not to be a standard _enough_ place to put those in the home folder.

Reply:
Just to be clear what I mean, this simply adds the string `source <(git-town completions bash)` to the end of the `.bashrc` file. It doesn't add the hundred lines of stuff that `git-town completions bash` outputs. Just this one line. With this in place, each time the user starts a new Bash shell, it runs `git-town completions bash` and evaluates them inside this shell, so that the autocompletions are active.